index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Atleti</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="//https://fonts.googleapis.com/css2?family=Outfit" />
+</head>
+<header class="main-header">
+
+    <div class="logo">
+        <img src="atletilogo.png" width="200" height="200" alt="AtletiLOGO">
+ 
+        <div class="sitename">
+            <p class="h1">Atleti.</p>
+        <p class="h2"><em>The number one source for the latest tennis news</em></p>
+            </div>
+ 
+    <div class="navbar">
+        <ul >
+            <li>
+                <a href="#">HOT TOPICS</a>
+            </li>
+            <li>
+                <a href="#">RESULTS</a>
+            </li>
+            <li>
+                <a href="#">QUIZZES</a>
+            </li>
+            <li>
+                <a href="#">GRAND SLAMS</a>
+            </li>
+            <li>
+                <a href="#">ATP TOUR 2022</a>
+            </li>
+            <li>
+                <a href="#">ARCHIEVE</a>
+            </li>
+        </ul>
+    </div>
+  
+</header>
+<article class="main-article">
+    <img class="article-pic" src="novak.jpeg" alt="Novak Djokivic" style="width:100%">
+    <div class="djokovic">
+        <p class="heading">AUSTRALIAN OPEN - Missing Novak Djokovic after historical vaccination mess. What's next for world tennis super star?</p>
```

Comment:
Could be a good practice to replace the `p `element with a `h` element if it's a heading.